src/krux/pages/file_manager.py

```diff
@@ -166,31 +166,51 @@ def display_file(self, file):
         """Display the file details on the device's screen"""
         import uos
         import time
+        from ..display import DEFAULT_PADDING, FONT_HEIGHT
+        from ..themes import theme
 
         stats = uos.stat(file)
         size_KB = stats[6] / 1024
         size_KB_fraction = str(int(size_KB * 100))[-2:]
         created = time.localtime(stats[9])
         modified = time.localtime(stats[8])
-        format_datetime = " %s-%02d-%02d %02d:%02d"
+        format_datetime = "%s-%02d-%02d %02d:%02d"
         file = file[4:]  # remove "/sd/" prefix
 
         self.ctx.display.clear()
-        self.ctx.display.draw_hcentered_text(
-            file
-            + "\n\n"
-            + t("Size:")
-            + " "
-            + generate_thousands_separator(int(size_KB))
-            + render_decimal_separator()
-            + size_KB_fraction
-            + " KB"
-            + "\n\n"
-            + t("Created:")
-            + format_datetime % created[:5]
-            + "\n\n"
-            + t("Modified:")
-            + format_datetime % modified[:5]
+        offset_y = DEFAULT_PADDING
+        offset_y += (
+            self.ctx.display.draw_hcentered_text(
+                file
+                + "\n\n"
+                + t("Size:")
+                + " "
+                + generate_thousands_separator(int(size_KB))
+                + render_decimal_separator()
+                + size_KB_fraction
+                + " KB"
+                + "\n\n",
```

Comment:
instead or printing new lines at the end, you can just increment offset with `2 * FONT_HEIGHT`